Lib/fontTools/ttLib/tables/_c_m_a_p.py

```diff
@@ -38,6 +38,21 @@ def getcmap(self, platformID, platEncID):
 				return subtable
 		return None # not found
 
+	def getBestCmap(self, cmapPreferences=((3, 10), (0, 4), (3, 1), (0, 3))):
+		"""Return the 'best' unicode cmap dictionary available in the font,
+		or None, if no unicode cmap subtable is available.
+
+		By default it will search for the following (platformID, platEncID)
+		pairs:
+			(3, 10), (0, 4), (3, 1), (0, 3)
+		But this can be customized via the cmapPreferences argument.
+		"""
+		for platformID, platEncID in cmapPreferences:
+			cmapSubtable = self.getcmap(platformID, platEncID)
+			if cmapSubtable is not None:
+				return cmapSubtable.cmap
+		return None  # None of the requested cmap subtables were found
```

Comment:
even without this line `return None` the function would have returned None. But explicit is better than implicit! :)

Reply:
I know, and yes, exactly! I always write `return None` if it has an explicit meaning other than "we're falling out of the method".